### ISSUE TITLE
Cannot add scroll listener to coast controller

### ISSUE THREAD

**Issue by mustafa-fahimi** [spkersten/flutter_coast#9]
Is there any way to detect when the scroll is finished or adding a scroll listener?

**Comment by spkersten** [comment]
You could try using a `CoastObserver`. Its `startTransition` will be call when a transition starts and its `animation` parameter tracks the progress; you can check the status of the animation to see if the transition was completed.

**Comment by mustafa-fahimi** [comment]
Can you provide an example inside the package example, please? Because it's very common to use.

**Comment by Gene-Dana** [comment]
Yeah it would be nice to see exactly how this works so I can figure out how to use it !
